project/MicroServiceBuild.scala

```diff
@@ -11,35 +11,38 @@ private object AppDependencies {
   import play.sbt.PlayImport._
 
 
-  private val playMicroServiceBootStrapVersion = "10.6.0"
+  private val bootstrapPlayVersion = "3.1.0"
   private val metricsGraphiteVersion = "3.0.2"
-  private val httpCachingVersion = "7.0.0"
+  private val httpCachingVersion = "9.1.0-play-26"
   private val bulkEntityStreamingVersion = "1.0.0"
-  private val domainVersion = "5.3.0"
+  private val domainVersion = "5.10.0-play-26"
   private val tabularDataValidatorVersion = "1.0.0"
   private val scalaXmlVersion = "1.0.6"
   private val scalaParserCombinatorsVersion = "1.0.6"
-  private val scalaTestVersion = "3.0.4"
-  private val scalaTestPlusPlayVersion = "2.0.1"
+  private val scalaTestVersion = "3.0.9"
+  private val scalaTestPlusPlayVersion = "3.1.3"
   private val pegdownVersion = "1.6.0"
-  private val jsoupVersion = "1.10.3"
+  private val jsoupVersion = "1.13.1"
   private val hmrcTestVersion = "3.3.0"
-  private val wiremockVersion = "1.58"
-  private val cryptoVersion = "4.4.0"
-  private val mockitoCoreVerison = "1.10.19"
+  private val wiremockVersion = "2.26.3"
+  private val cryptoVersion = "5.6.0"
```

Comment:
This is much harder to read than if they were just on the same line as the thing their version corresponds to.